Sources/Memberlist.php

```diff
@@ -304,7 +304,11 @@ function MLAll()
 	if ($use_cache && $_REQUEST['sort'] === 'real_name' && !isset($_REQUEST['desc']))
 	{
 		$first_offset = $_REQUEST['start'] - ($_REQUEST['start'] % $cache_step_size);
+		if ($first_offset < 0)
+			$first_offset = 0;
```

Comment:
```suggestion
		$first_offset = max(0, $_REQUEST['start'] - ($_REQUEST['start'] % $cache_step_size));
```

Reply:
Cleaner?  I disagree.  It is much easier to read what it's doing with the comparison to 0...

Reply:
_De gustibis non est disputandum_, I suppose. I personally find min() and max() to be clear and concise, and I prefer to avoid unnecessary `if` statements when I can. But if you have strong feelings about this, I won't dig my heels in. :)

Reply:
I think it's "gustibus"?

Not strong feelings, just don't like unnecessary changes.  Especially the requisite retests.  (And I honestly think it's less readable...  😉)

I admit my OCD side wants to redo it...  Although these range checks work, in a way, they just mask the real issues with the calc.  I think there are multiple problems: I suspect an off-by-one issue which is why the ceiling sometimes goes too high, DESC sorts have a different count of names on the last (now first...) page, & it fails pretty bad with large page sizes.   

But...  It works.   And the new range checks prevent it from erroring out no matter what.  And there are very few reports of these errors on the support forum anyway.  

I.e., it's a nit...